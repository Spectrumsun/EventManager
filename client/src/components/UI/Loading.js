import React from 'react';

const Loading = () => (
  <div className="container" style={{ paddingTop: '100px' }}>
    <div className="loader" />
    <p className="center-item shadow">
      Unable to connect Reaource Not Found or invalid Url.
        Refresh your browser or check your internet connection
    </p>
    <div className="container" style={{ paddingTop: '200px' }} />
  </div>
);

export default Loading;

