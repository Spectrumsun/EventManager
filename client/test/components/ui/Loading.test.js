import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../../../src/components/UI/Loading';

describe('<Home />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Loading />);
  });

  it('should render the HomePage', () => {
    shallow(<Loading />);
  });

  it('should render initial layout of HomePage', () => {
    expect(wrapper.getElements()).toMatchSnapshot();
  });
  
  it('should have three div on layout', () => {
    expect(wrapper.find('div').length).toEqual(3);
  });

  it('should have one image on layout', () => {
    expect(wrapper.find('p').length).toEqual(1);
  });
});
