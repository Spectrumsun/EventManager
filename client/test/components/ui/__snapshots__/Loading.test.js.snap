// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render initial layout of HomePage 1`] = `
Array [
  <div
    className="container"
    style={
      Object {
        "paddingTop": "100px",
      }
    }
  >
    <div
      className="loader"
    />
    <p
      className="center-item shadow"
    >
      Unable to connect Resource Not Found or invalid Url. Refresh your browser or check your internet connection
    </p>
    <div
      className="container"
      style={
        Object {
          "paddingTop": "200px",
        }
      }
    />
  </div>,
]
`;
